runtime/polkadot/src/lib.rs

```diff
@@ -1063,11 +1056,44 @@ pub type Executive = frame_executive::Executive<
 	frame_system::ChainContext<Runtime>,
 	Runtime,
 	AllModules,
-	()
+	(FixPolkadotCouncilVotersDeposit),
 >;
 /// The payload being signed in transactions.
 pub type SignedPayload = generic::SignedPayload<Call, SignedExtra>;
 
+struct FixPolkadotCouncilVotersDeposit;
+impl frame_support::traits::OnRuntimeUpgrade for FixPolkadotCouncilVotersDeposit {
+	fn on_runtime_upgrade() -> Weight {
+		use pallet_elections_phragmen::Voter;
+		use frame_support::IterableStorageMap;
+		let mut updated = 0;
+		let mut skipped = 0;
+		pallet_elections_phragmen::Voting::<Runtime>::translate::<Voter<AccountId, Balance>, _>(
+			|_who, mut vote| {
+				if vote.deposit == 5 * CENTS {
+					// If their deposit is what we set by mistake
+					vote.deposit = 5 * DOLLARS;
```

Comment:
just to be sure CENTS hasn't changed since the wrong migration, and DOLLARS hasn't change since they reserved, isn't it ?

Reply:
I should double check that indeed.

Reply:
Line has not been changed for 10 months 


<img width="1066" alt="Screenshot 2021-03-17 at 12 24 32" src="https://user-images.githubusercontent.com/5588131/111460002-5aa7b200-8713-11eb-9430-8bece059a547.png">